plugins/modules/nfs_exports_info.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/python
+
+# SPDX-FileCopyrightText: (c) 2025, Samaneh Yousefnezhad <s-yousefenzhad@um.ac.ir>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: nfs_exports_info
+
+short_description: Extract folders, IPs, and options from C(/etc/exports)
+
+description:
+  - This module retrieves and processes the contents of the C(/etc/exports) file from a remote server,
+    mapping folders to their corresponding IP addresses and access options.
+
+author:
+  - Samaneh Yousefnezhad (@yousefenzhad)
+version_added: "11.1.0"
```

Comment:
The next release would be 12.2.0:
```suggestion
version_added: "12.2.0"
```